### ISSUE TITLE
Invisible cursor in spacer for Voids

### ISSUE THREAD

**Issue by kib357** [ianstormtaylor/slate#574]
When i use Inline blocks with "isVoid": true, Slate renders invisible spacer before element. Its invisible, and when selection collapsed is in spacer, cursor also invisible:
https://monosnap.com/file/7SkqoxCtJscbUZM0Xxb5G5ioYmuHLZ
When i remove styles from spacer, all works fine, but cursor become visible, when it stays in Void element. Maybe spacer needs to be invisible only when Void is selected?

**Comment by kib357** [comment]
I don't know why, but this change in `void.js` fixes cursor:
```
if (node.kind == 'block') {
      style = IS_FIREFOX
        ? {
          pointerEvents: 'none',
          width: '0px',
          height: '0px',
          lineHeight: '0px',
          visibility: 'hidden'
        }
        : {
          position: 'absolute',
          top: '0px',
          left: '-9999px',
          textIndent: '-9999px'
        }
    } else {
      style = {
        position: 'relative',
        top: '0px',
      }
//
      if (state.selection.hasFocusIn(node)) {
        style.left = '-9999px'
        style.textIndent = '-9999px'
      }
//
    }
```

**Comment by kib357** [comment]
And also it's fixed in 0.16.4, so closing...
